apps/site/src/app/about/page.tsx

```diff
@@ -47,8 +47,7 @@ export default async function Home() {
 						</p>
 					</div>
 					<div className={clsx(styles.aboutTeams, "container")}>
-						{Object.entries(teamMembers).map(([team, members]) => (
-							// @ts-expect-error
+						{Object.entries(teamMembers!).map(([team, members]) => (
```

Comment:
Using a non-null assertion here probably isn't the best. `teamMembers` will be undefined if the sanity query or parsing errors. Perhaps throw an error on L22 if `teamMembers` is undefined?